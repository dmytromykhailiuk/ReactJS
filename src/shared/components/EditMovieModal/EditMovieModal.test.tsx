import { shallow } from "enzyme";
import React from "react";
import { EditMovieModal } from "./";
import toJson from 'enzyme-to-json';

describe("EditMovieModal", () => {
  it("should match snepshot", () => {
    const wrapper = shallow(
      <EditMovieModal
        movie={{} as any}
        onCloseModal={() => {}}
        onCloseWithSaving={() => {}}
      />
    )
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})