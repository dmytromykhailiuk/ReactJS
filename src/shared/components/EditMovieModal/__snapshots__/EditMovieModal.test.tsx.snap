// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditMovieModal should match snepshot 1`] = `
<Component
  header="EDIT MOVIE"
  onCloseModal={[Function]}
>
  <MovieForm
    movie={Object {}}
    onSubmitForm={[Function]}
  />
</Component>
`;
